frontend/src/app.tsx

```diff
@@ -11,16 +11,20 @@ import { Helmet } from 'react-helmet';
 import { registerCopyHandler } from './utils/copy_text';
 import { useAuthData } from './utils/auth';
 import { LoadingPage } from './pages/loading';
+import { PagePage } from './pages/page';
+import { AboutPage } from './pages/about';
 
 registerCopyHandler();
 
+const LOCATIONS_WIHTOUT_AUTH = ['/about'];
+
 export function App() {
   const routerBase = trimTrailingSlash(import.meta.env.BASE_URL);
 
-  const [_location, navigate] = useLocation();
+  const [location, navigate] = useLocation();
   const { isLoading, isLoggedIn, hasShareToken } = useAuthData();
   const isAuthenticated = isLoggedIn || hasShareToken;
-  if (!isAuthenticated && !isLoading) {
+  if (!isAuthenticated && !isLoading && !LOCATIONS_WIHTOUT_AUTH.includes(location)) {
     setTimeout(() => navigate('/login'), 0);
```

Comment:
I don't understand what you mean by "stop rendering the authenticated pages here"? We prevent the rendering using the optional routes later in the document. I agree that we could move this check to the respective pages, however I'd prefer to open an issue for that and do this in a seperate PR, as this might involve a bit more logic changes

Reply:
e. g. `return null` but looks fine with the `isLoggedIn` condition below. I'm also fine with the separate for the moved check.